### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Praffesor**

### CODE REVIEW

File: lib/rademade_admin/version.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # -*- encoding : utf-8 -*-
 module RademadeAdmin
-  VERSION = '0.3.0'
+  VERSION = '0.3.1'
```
Comment (Rademade-review) [inline lib/rademade_admin/version.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+          STRING = 1
+          BOOLEAN = 2
+          DATE = 3
+          DATE_TIME = 4
+
+        end
+      end
+    end
+  end
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# -*- encoding : utf-8 -*-
+module RademadeAdmin
+  module Model
+    class Info
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+          STRING = 1
+          BOOLEAN = 2
+          DATE = 3
+          DATE_TIME = 4
+
+        end
+      end
+    end
+  end
+end
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# -*- encoding : utf-8 -*-
+module RademadeAdmin
+  module Model
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+      class Field
+        module Type
+
+          STRING = 1
+          BOOLEAN = 2
+          DATE = 3
+          DATE_TIME = 4
+
+        end
+      end
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# -*- encoding : utf-8 -*-
+module RademadeAdmin
+  module Model
+    class Info
+      class Field
+        module Type
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/info/field/type.rb [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+          STRING = 1
+          BOOLEAN = 2
+          DATE = 3
+          DATE_TIME = 4
+
+        end
+      end
+    end
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# -*- encoding : utf-8 -*-
+module RademadeAdmin
+  module Model
+    class Info
+      class Field
```
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/configuration.rb [hunk 10]
```diff
@@ -101,4 +101,4 @@ def form(*field_options, &block)
 
     end
   end
-end
+end
```
Comment (Rademade-review) [inline lib/rademade_admin/model/configuration.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 2 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 3 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 4 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 5 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 6 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 7 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 8 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 9 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/configuration.rb (hunk 10 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/configuration.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/adapter/data/sequel.rb [hunk 11]
```diff
@@ -68,6 +66,23 @@ def _to_many_setter(relation_info, getter)
...
+            case type
+              when :string
+                RademadeAdmin::Model::Info::Field::Type::STRING
+              when :boolean
+                RademadeAdmin::Model::Info::Field::Type::BOOLEAN
+              when :date
+                RademadeAdmin::Model::Info::Field::Type::DATE
+              when :dateTime
+                RademadeAdmin::Model::Info::Field::Type::DATE_TIME
+              else
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/sequel.rb]: Redundant empty `else`-clause.

File: lib/rademade_admin/model/info/field/type.rb (hunk 2 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 3 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 4 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 5 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 6 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 7 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 8 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 9 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/configuration.rb (hunk 10 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/configuration.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/adapter/data/sequel.rb (hunk 11 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/sequel.rb]: Redundant empty `else`-clause.

File: lib/rademade_admin/model/info/field/type.rb (hunk 2 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 3 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 4 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 5 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at module body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 6 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 7 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/info/field/type.rb (hunk 8 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body end.

File: lib/rademade_admin/model/info/field/type.rb (hunk 9 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/info/field/type.rb]: Empty line missing at class body beginning.

File: lib/rademade_admin/model/configuration.rb (hunk 10 above)
Comment (Rademade-review) [inline lib/rademade_admin/model/configuration.rb]: Empty line missing at module body end.

File: lib/rademade_admin/model/adapter/data/sequel.rb [hunk 12]
```diff
@@ -68,6 +66,21 @@ def _to_many_setter(relation_info, getter)
...
+
+          def field_type(type)
+            case type
+              when :string
+                RademadeAdmin::Model::Info::Field::Type::STRING
+              when :boolean
+                RademadeAdmin::Model::Info::Field::Type::BOOLEAN
+              when :date
+                RademadeAdmin::Model::Info::Field::Type::DATE
+              when :dateTime
```
Comment (Rademade-review) [inline lib/rademade_admin/model/adapter/data/sequel.rb]: Indent `when` as deep as `case`.

**Comment by Melanitski** [comment]
@Praffesor можем мержить, но есть конфликты

**Pull request merged**